mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.tmpl

```diff
@@ -918,6 +918,28 @@ is set to true. Defaults to ZONAL.`,
 				},
 				Description: `The replicas of the instance.`,
 			},
+			"replication_cluster": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"failover_dr_replica_name": {
+							Type:        schema.TypeString,
+							Computed:    true,
```

Comment:
I'm not familiar with terraform, I marked this field as computed because of following: Google backend can normalize user-provided input for "failover_dr_replica_name", so input from user may not be the actual value after we apply the change.

If it's OK to skip computed on this scenario, I'll remove it.

Reply:
If there is a default value from API is returned when no value if provided by the user, then `Computed: true` can be used to suppress the diffs. 

What `Computed: true` does is that if no value is provided by the user in the Terraform configuration, its previous value from API is used instead. 

We can try to remove `Computed: true` now.

Reply:
Do you have the logs that how backend normalizes the user-provided input for `ailover_dr_replica_name` ?

This is the [doc](https://googlecloudplatform.github.io/magic-modules/develop/diffs/#normalized-value) how to handle this case.